src/NonlinearArithmetic/DivMod.dfy

```diff
@@ -209,7 +209,7 @@ module {:options "-functionSyntax:4"} DivMod {
     ensures forall a: int, b: int, d: int, R: int {:trigger d * ((a + b) / d) - R, d*(a/d) + d*(b/d)}
               :: 0 < d &&  R == a%d + b%d - (a+b)%d ==> d*((a+b)/d) - R == d*(a/d) + d*(b/d)
   {
-    forall (a: int, b: int, d: int, R: int {:trigger true} | 0< d &&  R == a%d + b%d - (a+b)%d)
+    forall (a: int, b: int, d: int, R: int {:trigger d * ((a + b) / d) - R, d*(a/d) + d*(b/d)} | 0< d &&  R == a%d + b%d - (a+b)%d)
```

Comment:
I think you can't use interpreted functions, like any of the arithmetic or logical operators, in a trigger. Sorry that there is no automatic feedback on this, or even good documentation. And to be honest I'm not sure what the right trigger for this forall would be. @MikaelMayer @atomb ?

Reply:
You can run Dafny and it will give you some help, but also `{:nowarn}` works in newer versions of Dafny, and that gets me what I want!

Reply:
Please use `{:nowarn}` everywhere then. These triggers are not valid triggers because they map to builtin operators that do not work with pattern matching.